python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -399,8 +399,10 @@ def run_transformer_int4_gpu(repo_id,
             # in_len.txt maybe shorter than we need,
             # use much longer context to make sure input length
             test_length = min(in_len*2, 8192)
-            while test_length not in [32, 256, 1024, 2048, 8192]:
+            prompt_list = [32, 256, 1024, 2048, 8192]
+            while test_length not in prompt_list and test_length < max(prompt_list):
```

Comment:
just `while test_length not in [32, 256, 1024, 2048, 8192] and test_length < 8192:`?

Reply:
ok